src/cartesian/Area.tsx

```diff
@@ -120,7 +131,20 @@ class Area extends PureComponent {
   };
 
   static getComposedData = ({ props, xAxis, yAxis, xAxisTicks, yAxisTicks, bandSize,
-    dataKey, stackedData, dataStartIndex, displayedData, offset }) => {
+    dataKey, stackedData, dataStartIndex, displayedData, offset }: {
+      props: Props;
+      item: Area;
+      bandSize: number;
+      xAxis: InternalAreaProps['xAxis'];
+      yAxis: InternalAreaProps['yAxis'];
+      xAxisTicks: XAxisProps['ticks'];
+      yAxisTicks: YAxisProps['ticks'];
```

Comment:
I also see `XAxisProps['ticks']` pattern a lot. What do you think is better, `XAxisProps['ticks']` or `TickItem[]`?